### DISCUSSION TITLE
Laravel →green() macro to quickly set the colors

### DISCUSSION THREAD

**Discussion by Gompje** [spatie/ray#95] [Ideas]
Instead of setting a color with a string, some defaults should be great like →green() to log in green. or even →warning(),..

**Comment by AdrianMrn** [comment]
We did have a short discussion about this when implementing the colors, but we ended up going for the unbiased approach and letting people pick their own reasons for adding a certain color.

Great suggestion and thanks a lot for thinking along with us!

**Reply by Gompje** [reply]
I will make my own small layer :) works for me!
